src/renderer/auth/components/register/VerifyAccount/VerifyAccount.js

```diff
@@ -105,8 +99,13 @@ export default class VerifyAccount extends React.PureComponent {
     );
   };
 
-  renderPrevious = () => {
-    console.log('dojawijdoaiwjdaow');
-    return <RegisterForm />;
+  onBack = () => {
+    this.props.setStep(2);
```

Comment:
Perhaps `setStep` could be dynamically set here e.g. `currentStep - 1`

Reply:
Good point! I was thinking the same, wanted to refactor it - but didn't get it yet